### ISSUE TITLE
About training detais of Context-FID score and other measures 

### ISSUE THREAD

**Issue by JinseongP** [Y-debug-sys/Diffusion-TS#13]
Hi, thank you for your kind words and for noting the ease of reproducing your code.

When I ran all the configuration files in your code, I encountered serious problems without altering any part of it.

After training and sampling energy data,

![image](https://github.com/Y-debug-sys/Diffusion-TS/assets/53927724/ddf96e6d-2ae5-4d76-b667-0e9a34f3e450)

the results of the measures have significantly different scales in your code.

![image](https://github.com/Y-debug-sys/Diffusion-TS/assets/53927724/2c950e13-b8ea-483e-8923-629ecbe30668)

(1) From my perspective, this misalignment could have occurred due to the training code in TS2Vec, possibly requiring adjustments to the learning rate or batch size. (maybe the code is fitted to Sine data.)

Did you use a different training approach, such as lr or batch size,  for TS2Vec in the following code?

![image](https://github.com/Y-debug-sys/Diffusion-TS/assets/53927724/0f8957ed-d490-4c57-a8de-0fcb4671a4b2)

or should I apply some scaling techniques before calculating the evaluation scores?

(2) In the discriminative score and predictive score, did you also change the hyperparameters, such as the learning rate or batch size to train discriminator or predictor?

Please kindly correct me if I have any misunderstanding of your code. Thanks.

**Comment by Y-debug-sys** [comment]
Some issues have reported the same question due to my negligence, I will provide detailed explanations of some precautions soon. The reason is that the Sine dataset used for the demonstration, its ori_data is saved as an npy file with the name containing 'ground_truth,' but for other datasets, it should have names containing 'norm_truth' in the npy file. Now uncomment the code in .iqynb file, then rerun it please.
